cellrank/external/kernels/_wot_kernel.py

```diff
@@ -338,6 +338,15 @@ def compute_transition_matrix(
         kwargs["growth_iters"] = max(growth_iters, 1)
         self_transitions = SelfTransitions(self_transitions)
 
+        if (
+            (self_transitions is not None)
```

Comment:
This check only makes sense when `self_transitions == 'all'` or is a tuple (`'uniform'/'diagonal'/'connectivities'` don't need the `conn_weight`).
Please change this check.